app/views/diary_entries/_diary_entry.html.erb

```diff
@@ -1,3 +1,7 @@
+<% content_for :head do %>
+  <%= javascript_include_tag "social_share_button" %>
```

Comment:
You'll have as many `<script>` tags as there are articles and a bunch of js errors about redeclarations.

Reply:
Added this into `_head.html.erb`